swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/AbstractResourceTypeHandler.kt

```diff
@@ -108,10 +102,50 @@ abstract class AbstractResourceTypeHandler<T : Resource>(
   }
 
   /**
-   * Notifies resource owners
+   * Notifies for everything in resourceRepository.getMarkedResources() that doesn't have notificationInfo
    */
   override fun notify(workConfiguration: WorkConfiguration) {
```

Comment:
This now schedules notifications when enabled, otherwise update the resource notification info so it can be ready for deletion. AMIs and Snapshots would fall in this category

Reply:
do we have any tests to validate this for AMI's and snapshots ?

Reply:
Please elaborate

Reply:
Ignore , i saw that you have the tests based on `resourceType` in `NotificationSenderTest`